migrations/20180630155901_users.js

```diff
@@ -1,14 +1,19 @@
-
-exports.up = function(knex, Promise) {
-  return knex.schema.createTable('users', (table) => {
-    table.increments()
-    table.string('name').notNullable()
-    table.string('email').unique().notNullable();
-    table.specificType('hashed_password', 'char(60)').notNullable();
-    table.timestamps(true, true)
-  })
-}
-
-exports.down = function(knex, Promise) {
-  return knex.schema.dropTable('users')
-}
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('users', (table) => {
+    table.increments()
+    table.string('name').notNullable()
+    table.string('email').unique().notNullable();
+    table.specificType('hashed_password', 'char(60)').notNullable();
+    table.string('city').notNullable();
+    table.text('bio').notNullable();
+    table.text('image_url').notNullable();
```

Comment:
@Brechy @niksseif 
1. Should we have a default image url (such as https://cdn.stocksnap.io/img-thumbs/960w/XNMZNBEQHM.jpg) populate in the database OR just have a static image that is pulled up automatically if image_url is null (would allow it to be nullable in that case).
2. Any ideas for default image to use?